admin/sql/CreateTables.sql

```diff
@@ -21,6 +21,7 @@ CREATE TABLE application
 
 CREATE TABLE area_type ( -- replicate
     id                  SERIAL, -- PK
+    gid                 uuid NOT NULL,
```

Comment:
I'm not sure this would practically cause us any problems, but since `ALTER TABLE` in the upgrade script will add the column to the end, I wonder if we should do the same here for consistency.